docs: API link to object (#394)
akka/akka-projection: docs/src/main/paradox/getting-started/projection-handler.md

Scala
:  @@snip [ShoppingCartApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartApp.scala) { #guideProjectionHandler }

> The projection is run by wrapping it in a @apidoc[ProjectionBehavior$] and running it with a typed @apidoc[akka.actor.typed.ActorSystem].

Reviewer: ```suggestion
The projection is run by wrapping it in a @apidoc[ProjectionBehavior$] and spawning it as an Actor in the @apidoc[akka.actor.typed.ActorSystem].
```

Reviewer: I have changed the same line in https://github.com/akka/akka-projection/pull/387/files#diff-a1a7a3d5dfaea940b8878188ee8ebf00R46 so I changed to `ProjectionBehavior$` there also to avoid merge conflict.
